src/resolvers/offer/offer_authorization.ts

```diff
@@ -0,0 +1,56 @@
+import { ApolloError, ForbiddenError } from 'apollo-server-express'
+import { AuthenticatedContext } from '../../apolloServer'
+import Offer from '../../models/offer'
+import { OfferStatus, ShipmentStatus } from '../../server-internal-types'
+
+const assertAssociations = (offer: Offer): void => {
```

Comment:
Suggestion (non-blocking): some JSDoc-style comments for these methods would be very helpful!

Reply:
done!